### PR TITLE
tutorial part for complete entity

### PR DESCRIPTION
**Pull request by patriknw**

### CODE REVIEW

File: docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc [hunk 1]
```diff
@@ -5,21 +5,117 @@
...
+* `Checkout` - checkout the shopping cart
+* `Get` - get the current state of the shopping cart
+
+and corresponding events:
+
+* `ItemRemoved`
+* `ItemQuantityAdjusted`
+* `CheckedOut`
+
+Adding those commands, events and adjusting the `State` is very similar to the `AddItem` command and `ItemAdded` event in the xref:tutorial/entity.adoc[previous step], which means that this is a good exercise for you to solidify your knowledge of how to implement an `EventSourcedBehavior`. If you like, you can start with <<Expand the unit test>> for a test driven approach.
```
Comment (chbatey) [inline docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:29]: Is the <<Expand >> some magic i don't know about?

**Review by chbatey — APPROVED** [review]

File: docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc (hunk 1 above)
Comment (patriknw) [reply docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:29]: links within same page, https://github.com/lightbend/cloudstate-docs/blob/master/asciidoc-antora-ovr.md

File: docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc [hunk 2]
```diff
@@ -5,21 +5,117 @@
 
 include::ROOT:partial$include.adoc[]
 
+In this part we will complete the `ShoppingCart` entity with more commands and events. It is using Event Sourcing to persist events that represents changes to the state of the cart.
```
Comment (ennru) [inline docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:8]:
```suggestion
In this part, we will complete the `ShoppingCart` entity with more commands and events. Event Sourcing persists these events which represent changes to the cart state.
```

File: docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc [hunk 3]
```diff
@@ -5,21 +5,117 @@
...
+
+and corresponding events:
+
+* `ItemRemoved`
+* `ItemQuantityAdjusted`
+* `CheckedOut`
+
+Adding those commands, events and adjusting the `State` is very similar to the `AddItem` command and `ItemAdded` event in the xref:tutorial/entity.adoc[previous step], which means that this is a good exercise for you to solidify your knowledge of how to implement an `EventSourcedBehavior`. If you like, you can start with <<Expand the unit test>> for a test driven approach.
+
+When you have completed your code you can compare with:
```
Comment (ennru) [inline docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:31]:
```suggestion
When you have completed your code you can compare it with:
```

**Review by ennru — APPROVED** [review]
LGTM, while doing exercises from the technical writing course...

File: docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc (hunk 2 above)
Comment (patriknw) [reply docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:8]: not sure about that change. It refers to the ShoppingCart. Maybe that can be clarified by spelling it out
"The `ShoppingCart` is using event sourcing..."

that vs which? I think that part is rather essential for the sentence, i.e. "that" https://developers.google.com/tech-writing/one/short-sentences

Comment (patriknw) [reply docs-source/docs/modules/cqrs/pages/tutorial/complete-entity.adoc:8]:
```suggestion
In this part, we will complete the `ShoppingCart` entity with more commands and events. The `ShoppingCart` is using Event Sourcing to persist events that represents changes to the state of the cart.
```

**Pull request merged**